app/containers/ErrorBoundary/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ *
+ * ErrorBoundary
+ *
+ */
+
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { translationMessages } from 'i18n';
+import { createStructuredSelector } from 'reselect';
+import { makeSelectLocale } from 'containers/LanguageProvider/selectors';
+
+import messages from './messages';
+import ErrorBoundaryPage from './ErrorBoundaryPage';
+
+/* eslint-disable react/prefer-stateless-function */
+export class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      errorInfo: null,
+    };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    this.setState({
+      error,
+      errorInfo,
+    });
+  }
+
+  reloadApp = () => {
+    document.location.href = '/';
```

Comment:
how that approach is different from `window.location.reload()`?
I would prefer to have .js file in utils with function reloads page. This way it can be reused and mocked in unit tests if needed.